welcome-message/main.scss

```diff
@@ -1,15 +1,16 @@
 @import 'o-visual-effects/main';
+@import 'o-icons/main';
 
 @if nUiApply('welcomeMessage') {
 
 	@include nUiMaybeCriticalStart('welcomeMessage');
 
-	.n-banner-message,
+	.n-welcome-banner,
 	.n-welcome-message {
 		position: relative;
 		padding: 16px 12px;
 		// not in o-colors ಠ_ಠ
-		background: #FADBCB;
+		background: #fadbcb;
```

Comment:
any chance we could swap this out to something acceptably close that is in o-colors?